plugins/tpm/README.md

```diff
@@ -0,0 +1,26 @@
+TPM Support
+===========
+
+Introduction
+------------
+
+This allows enumerating Trusted Platform Modules, also known as "TPM" devices,
+although it does not allow the user to update the firmware on them.
+
+The TPM Event Log is also available, which may help in explaining why PCR0
+values are differing for some firmware.
+
+GUID Generation
+---------------
+
+These devices use custom GUIDs:
+
+
+ * `TPM\VEN_$(vendor)&DEV_$(type)`
+ * `TPM\VEN_$(vendor)&DEV_$(type)_VER_$(family)`, where `family` is either `2.0` or `1.2`
+
+Example GUIDs from a real system containing a TPM from Intel:
```

Comment:
You mean firmware TPM?  Intel doesn't make discrete TPMs

Reply:
I guess I might have a "soft" TPM then, but my vendor is clearly `INT` and the string is "Intel".